CHANGELOG.md

```diff
@@ -4,15 +4,16 @@ X.Y.Z Release notes
 * None.
 
 ### Enhancements
-* None.
+* Added new query features.
```

Comment:
This should probably list the new query features. I believe they're listed in core's change log.